nowcasting_dataset/filesystem/utils.py

```diff
@@ -10,15 +10,24 @@
 _LOG = logging.getLogger("nowcasting_dataset")
 
 
-def upload_and_delete_local_files(dst_path: str, local_path: Path):
+def upload_and_delete_local_files(dst_path: Union[str, Path], local_path: Union[str, Path]):
     """
     Upload an entire folder and delete local files to either AWS or GCP
     """
     _LOG.info("Uploading!")
     filesystem = get_filesystem(dst_path)
-    filesystem.put(str(local_path), dst_path, recursive=True)
+
+    _LOG.warning(f"moving files from {local_path} to {dst_path}")
+
+    _LOG.warning(get_all_filenames_in_path(local_path))
```

Comment:
Please make these two messages a bit more verbose  (or maybe remove these two messages?)  Also, do these need to be warnings?